package/kedro_viz/models/utils.py

```diff
@@ -0,0 +1,25 @@
+"""`kedro_viz.models.utils` contains utility functions used in the `kedro_viz.models` package"""
```

Comment:
Very good innovation to make this file. There's quite a few candidates for other things that should be moved into this file at some point.

Reply:
I totally agree, I don't think it's worth a ticket, but it's something to keep in mind as we make more changes to `flowchart.py` and `experiment_tracking.py`.

Reply:
I totally agree, I don't think it's worth a ticket, but it's something to keep in mind as we make more changes to `flowchart.py` and `experiment_tracking.py`.

Reply:
I'll add it to the mega-ticket https://github.com/kedro-org/kedro-viz/issues/980 where I keep track of this sort of thing and will at some point break it down into more manageable tasks or chip away at it bit by bit. Looks like I'd already put de-duplicating the `get_dataset_type` code there so we're definitely on the same wavelength!
> * Reuse `DataNode` and `DataNoteMetadata` models. There's too much duplication between these and tracking datasets (also code to get dataset type is in both)